builder/dockerfile/internals.go

```diff
@@ -546,6 +554,62 @@ func hostConfigFromOptions(options *types.ImageBuildOptions) *container.HostConf
 	}
 }
 
+func throttleDeviceFromOptionWithSize(option string) []*blkiodev.ThrottleDevice {
+	configuration := strings.Split(option, ":")
+	if len(configuration) != 2 {
+		return nil
+	}
+	var rate int
+	var err error
+	if strings.HasSuffix(configuration[1], "kb") {
```

Comment:
This is basically repeating the functionality of [docker/go-units](https://github.com/docker/go-units), which is used by other similar places. Please use it here, too.